### PR TITLE
prometheus: Use TLS certs for tenant proxy

### PR DESCRIPTION
**Pull request by brancz**
Fixes what was reported in this comment: https://github.com/openshift/console/pull/909#issuecomment-452048328

@metalmatze @squat @mxinden @s-urbaniak 

cc @spadgett

### CODE REVIEW

File: assets/grafana/dashboard-definitions.yaml [hunk 1]
```diff
@@ -2026,7 +2026,7 @@ items:
                           "steppedLine": false,
                           "targets": [
                               {
-                                  "expr": "sum(max(node_filesystem_size_bytes{fstype=~\"ext[234]|btrfs|xfs|zfs\"} - node_filesystem_avail_bytes{fstype=~\"ext[234]|btrfs|xfs|zfs\"}) by (device,pod,namespace)) by (pod,namespace)\n/ scalar(sum(max(node_filesystem_size_bytes{fstype=~\"ext[234]|btrfs|xfs|zfs\"}) by (device,pod,namespace)))\n* on (namespace, pod) group_left (node) node_namespace_pod:kube_pod_info:\n",
+                                  "expr": "sum(max(node_filesystem_size_bytes{fstype=\u007e\"ext[234]|btrfs|xfs|zfs\"} - node_filesystem_avail_bytes{fstype=\u007e\"ext[234]|btrfs|xfs|zfs\"}) by (device,pod,namespace)) by (pod,namespace)\n/ scalar(sum(max(node_filesystem_size_bytes{fstype=\u007e\"ext[234]|btrfs|xfs|zfs\"}) by (device,pod,namespace)))\n* on (namespace, pod) group_left (node) node_namespace_pod:kube_pod_info:\n",
```
Comment (metalmatze) [inline assets/grafana/dashboard-definitions.yaml]: Does this work like this?

Comment (brancz) [reply assets/grafana/dashboard-definitions.yaml]: no idea how that even got there

**Comment by spadgett** [comment]
thanks @brancz

**Pull request merged**